plugins/net_plugin/net_plugin.cpp

```diff
@@ -910,6 +914,7 @@ namespace eosio {
       std::atomic<bool>               block_sync_throttling{false};
       std::atomic<std::chrono::nanoseconds>   last_bytes_sent{0ns};
       std::atomic<boost::asio::ip::port_type> remote_endpoint_port{0};
+      std::atomic<uint64_t>           invalid_finality_msg_total{0};
```

Comment:
This appears to not be used. It is only incremented and set to 0, never checked. Seems like the `block_status_monitor` is doing everything you need.
We may decide we want to increase `max_consecutive_rejected_windows_` if we find we see a lot of `vote_status::unknown_block`. Or we may wish to track them separately.

Reply:
https://github.com/AntelopeIO/leap/issues/1569 specifies to track the failures and indicates eventually we may decide to disconnect. The counter is implemented for that purpose. Will ask @arhag in today's meeting to clarify.

Reply:
Use block_status_monitor per discussion. Thanks.